SECURITY.md

```diff
@@ -0,0 +1,78 @@
+
+### When To Send A Report
+
+You think you have found a vulnerability in a Helm project or a dependency of a Helm project. This can be any of the repositories on the [Helm GitHub organization](https://github.com/helm).
+
+### When Not To Send A Report
+
+* If a vulnerability has been found in an application deployed by a Helm Chart. Instead, contact the application maintainers
+* For guidance on securing Helm. Please see the documentation or ask in one of the support channels
+* You are looking for Helm applying security updates
+
+### Security Vulnerability Response
+
+Each report will be reviewed and receipt acknowledged within 3 business days. This will set off the security review process detailed below.
+
+Any vulnerability information shared with the security team stays within the Helm project and will not be shared with others unless it is necessary to fix the issue. Information is shared only on a need to know basis.
+
+As the security issue through triage, identification, and release the reporter of the security vulnerability will be notified. Additional questions about the vulnerability may also be asked of the reporter.
+
+### Public Disclosure
+
+A public disclosure of security vulnerabilities is released alongside release updates or details that fix the vulnerability. We try to fully disclose vulnerabilities once a mitigation strategy is available. Our goal is to perform a release and public disclosure quickly and in a timetable that works well for users. For example, a release may be ready on a Friday but for the sake of users may be delayed to a Monday.
+
+CVEs will be assigned to vulnerabilities. Due to the process and time it takes to obtain a CVE ID disclosures will happen first. Once the disclosure is public the process will begin to obtain a CVE ID. Once the ID has been assigned the disclosure will be updated.
+
+If the vulnerability reporter would like their name and details shared as part of the disclosure process we are happy to. We will ask permission and for the way the reporter would like to be identified. We appreciate vulnerability reports and would like to credit reporters if they would like the credit.
+
+## Security Team Membership
+
+The security team is made up of Helm project maintainers who are willing and able to respond to vulnerability reports.
+
+### Responsibilities
+
+* Members MUST be active project maintainers as defined in [the governance](https://github.com/helm/community/blob/master/governance/governance.md)
+* Members SHOULD engage in each reported vulnerability, at a minimum to make sure it is being handled
+* Members MUST keep the vulnerability details private and only share on a need to know basis
+
+### Membership
+
+New members are required to be active maintainers of Helm projects who are willing to perform the responsibilities outlined above. The security team is a subset of the maintainers. Members can step down at any time and may join at any time.
```

Comment:
Oh! Maybe this is sufficient to address my earlier concern with saying "subset of..."

Reply:
I'd rather be explicit all over the place. We too often read by skimming.